com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -1180,11 +1180,8 @@ internal void SynchronizeNetworkObjects(ulong clientId)
                 ClientSynchEventData.OnWrite(nonNullContext.NetworkWriter);
 
                 var size = bufferSizeCapture.StopMeasureSegment();
-                foreach (var sceneIndex in ClientSynchEventData.ScenesToSynchronize)
-                {
                     m_NetworkManager.NetworkMetrics.TrackSceneEventSent(
-                       clientId, (uint)ClientSynchEventData.SceneEventType, ScenesInBuild[(int)sceneIndex], size);
-                }
+                       clientId, (uint)ClientSynchEventData.SceneEventType, "", size);
```

Comment:
This change also fixes an issue where the total size of the message is reported for each scene, which makes it look like multiple messages were sent at that size when it is in fact one message that is sent about multiple scenes. With this change, we only report one metric so the size is accurate.